trulens_eval/examples/expositional/models/arctic_quickstart.ipynb

```diff
@@ -4,14 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Have we noted this missing functionality with the python api team?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Mostly it was mentioned in their doc https://docs.snowflake.com/user-guide/snowflake-cortex/llm-functions#using-snowflake-cortex-llm-functions-with-python (i.e. missing support for chat style input)